shared/js/dataDriver.js

```diff
@@ -172,6 +181,13 @@ loop.DataDriver = function() {
     }
 
     /**
+     * Get the initial retry delay, in ms.
+     */
+    get INITIAL_RETRY_TIMEOUT() {
+      return 2000;
+    }
+
```

Comment:
Hmm.. I thought I pointed this out earlier, but this should be defined between `BASE_FIREBASE` and `MAX_LIMIT`.